beginner.md

```diff
@@ -0,0 +1,588 @@
+# delite - creating custom components
+
+## delite background
+`delite` is a new JavaScript library born out of the [Dojo Toolkit Dijit framework](http://dojotoolkit.org/reference-guide/1.10/dijit).
```

Comment:
OK but that sounds more like a strategic decision not to mention Dojo or Dijit or the fact that most of the devs and likely a lot of ideas originated there. 
I'll look to re-word it